scripts/userstyles.yml

```
@@ -919,4 +920,10 @@ userstyles:
     color: red
     current-maintainers: [*isabelroses, *uncenter]
     past-maintainers: [*elkrien]
+  zen-browser:
```

It'd make my life easier if the key wasn't `zen-browser` since we already have a port that's catppuccin/zen-browser so I think appending `-docs` is fine

```suggestion
  zen-browser-docs:
```

---

Should I rename it everywhere, like the folder name (`styles/zen-browser-docs`), branch name (`feat/zen-browser-docs`), and PR title (`feat(zen-browser-docs): init`) ?

---

> Should I rename it everywhere, like the folder name (`styles/zen-browser-docs`), branch name (`feat/zen-browser-docs`), and PR title (`feat(zen-browser-docs): init`) ?

The folder must be renamed, the linter will complain otherwise. The branch name has no impact whatsoever, you could have named that wherever you wanted- no need to change it (it would make you close this PR anyway which is a pain ). And the PR title should be updated since that is used as the squash commit message, and should match the metadata.